spec/components/candidate_interface/decoupled_references_review_component_spec.rb

```diff
@@ -97,17 +97,28 @@
     end
   end
 
-  context 'when reference state is "cancelled" and the reference is complete' do
-    let(:feedback_requested) { create(:reference, :feedback_requested) }
-    let(:cancelled) { create(:reference, :cancelled) }
+  context 'when reference state is "cancelled" or "email_bounced" and the reference is complete' do
```

Comment:
The contents of this block are a little tricky to follow. Might it make sense to have separate contexts for `cancelled` and `email_bounced`? A side benefit is that then your `email_bounced` context doesn't have to state " and the reference is complete", which isn't relevant to the behaviour we want to test (ie - according to other specs in this file, we could never have sent it in the first place if it was incomplete).